test/sharness/status/results/status_new.md

```diff
@@ -0,0 +1,339 @@
+# IPFS Implementation Status
```

Comment:
Do we want to check this file in?

Reply:
I just checked it in so that people can see what a result looks like. But yeah I could have used a gist instead. I will remove it.

Reply:
This is the Gist that replaces the status_new.md file: https://gist.github.com/chriscool/1c07b5944a6fc058cf677a04fec62eec

I will change the PR description.

Reply:
Description and comment changed. Commit that removes the status_new.md file added.